src/store/functions/deleteComment.js

```diff
@@ -0,0 +1,7 @@
+import { deleteComment } from '../actions';
+
+const deleteCommentFromPost = ids => (dispatch) => {
```

Comment:
use shorthand for dispatching

Reply:
I don't understand what you mean, help, please

Reply:
https://react-redux.js.org/using-react-redux/connect-mapdispatch#defining-mapdispatchtoprops-as-an-object